### DISCUSSION TITLE
BT speaker

### DISCUSSION THREAD

**Discussion by skank01** [music-assistant/support#3273] [Q&A] [answered]
MA is installed as an addon here in HA :)
Is there a way i can play music to a Bluetooth speaker?

**Comment by OzGav** [comment, marked as answer]
https://music-assistant.io/faq/stream-to/

**Comment by skank01** [comment]
My HA and MA is running on a pi, i can sidedo that?
Or can i connect HA or MA to a bluetooth speaker?
